src/auth/auth.service.ts

```diff
@@ -0,0 +1,60 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+
+import { parseApiKeysConfig } from '~/auth/auth.utils';
+import { CreateApiKeyDto } from '~/auth/dto/create-api-key.dto';
+import { ApiKeyModel } from '~/auth/models/api-key.model';
+import { generateBase64Secret } from '~/common/utils';
+import { User, UserId } from '~/users/user.consts';
+
+@Injectable()
+export class AuthService {
+  private userStore: Record<UserId, User> = {};
+
+  constructor(config: ConfigService) {
```

Comment:
In other places we register our own dynamic config provider, but looking at tutorials it seems much more common to use the build in config service.

I think using the Nest service is better going forward and is much less boiler plate.